product_cost_automatic/__init__.py

```diff
@@ -0,0 +1,12 @@
+from odoo import SUPERUSER_ID, api
+
+from . import models
+
+
+def _set_is_automatically(cr, registry):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    products = env['product.product'].search([])
+    inactive_automatic_products = products.filtered(lambda x:x.standard_price > 0.0)
```

Comment:
We are filtering products those price have greater then zero, so for that we no need to set is_automatically. Because for that price already set. This executed only once when we install/upgrade module.